terraform-modules/storage-bucket/outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "bucket" {
```

Comment:
I am pretty sure if you specify the top-level resource then you have access to all the lower level variable that make up the resource.  so name and url I don't think are necessary as separate outputs

Reply:
yep ,  I actually reused the output from another module, will remove it